frontend/Cargo.toml

```diff
@@ -25,3 +25,5 @@ thiserror = { workspace = true }
 tracing = { workspace = true, default-features = false }
 time = { version = "0.3", features = ["wasm-bindgen"] }
 tracing-subscriber = { workspace = true, features = ["time"] }
+
+split-yew = { git = "https://github.com/aleb2000/split-yew", branch = "yew-playground" }
```

Comment:
Why not import from crates.io?

Reply:
If you want I can already address and update all dependencies to the crates.io version, however I am unsure what version of the monaco editor to use as you are also using a special branch that imports Yew from github.

Reply:
Since I saw you updated the dependencies on master, I did the same on this branch and it should now be resolved. I also published the updated version of split-yew so no bug anymore.

Marking this conversation as resolved.